packages/dashboard/src/contexts/DashContext/state.ts

```diff
@@ -99,6 +100,32 @@ export const reducer = (state: State, action: Action): State => {
       }
 
       return newState;
+    case "update-provider-message-sender":
+      const newProviderMessages = new Map(state.providerMessages);
+      const newSender = data;
```

Comment:
Not necessary but just an idea, we could think about namespacing things in `data`. In this case `data` would look like the following:
```
{
  address: "0x1234567890123456789012345678901234567890"
}
```

The benefit would just be that it would be a little more explicit.

Reply:
Blah I figured a while ago that [ts can tell me what `data` is](https://github.com/trufflesuite/truffle/blob/v5.7.3/packages/dashboard/src/contexts/DashContext/types/Action.ts) and yell at me if I get it wrong :)

Reply:
Yeah totally fine! I just like to get in the habit of throwing out suggestions for stylistic things in case it inspires something. I don't mind the way it is at all.